README.rst

```diff
@@ -1,8 +1,29 @@
 SignalFx was acquired by Splunk in October 2019. See https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html for more information.
 
+
+
 Python client library for SignalFx
 ==================================
 
+:warning:This repository and its published libraries are deprecated
```

Comment:
There's also a deprecation notice for signalflow [below](https://github.com/signalfx/signalfx-python/pull/142/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fL272). Not sure if we want to consolidate or mention it.

Reply:
Hey - I would leave it where it is. I think if someone is looking for signalflow they will find it in right place